docker-compose.yml

```diff
@@ -68,6 +76,14 @@ services:
     - ACCEPT_EULA=Y
     - SA_PASSWORD=Strong!Passw0rd
 
+  sqledge:
+    image: mcr.microsoft.com/azure-sql-edge:latest
```

Comment:
I'm curious, why do you need that? The SQL version we use for other tests is not compatible with ARM64?

Reply:
That’s right, the only sql server like available for arm64 is the azure sql edge, which listen to the same sql port and we can connect using the same sqlclient code.